package.json

```diff
@@ -10,7 +10,7 @@
     "build:dist": "NODE_ENV=production webpack --config webpack.config.production.js",
     "lint": "eslint --ext .js,.jsx src",
     "deploy": "npm run build:dist && surge -p .",
-    "start": "node server.js"
+    "start": "NODE_ENV=development node server.js"
```

Comment:
Side Note -- this will break windows compatibility. The `build:dist` is probably OK for breaking, but might be nice to have this still work. Can we just leave _no_ `NODE_ENV` and have that be defaulted to be `development`?

(or, use `builder envs` ;) )